lldb/test/API/functionalities/process_save_core_minidump/TestProcessSaveCoreMinidump.py

```diff
@@ -62,6 +63,17 @@ def verify_core_file(
             # Try to read just past the red zone and fail
             process.ReadMemory(sp - red_zone - 1, 1, error)
             self.assertTrue(error.Fail(), "No failure when reading past the red zone")
+            # Verify the registers are the same
+            self.assertTrue(thread_id, stacks_to_registers_map)
```

Comment:
I am not sure what is `self.assertTrue` here for. Do you mean `assertIn`?

Reply:
Yep, that was an overlook on my part.